libs/sdk-ui-ext/src/internal/components/configurationControls/OrientationDropdownControl.tsx

```diff
@@ -18,9 +18,53 @@ export interface IOrientationDropdownControl {
     pushData: (data: IPushData) => void;
 }
 
+export function getAxisesByChartOrientation(properties: IVisualizationProperties) {
+    const { xaxis, yaxis } = properties?.controls || {};
+    const isInverted = (properties?.controls?.orientation?.position as ChartOrientationType) === "vertical";
```

Comment:
please move this line below the next if statement, so that it would be like
```
    if (!xaxis && !yaxis) {
        return { xaxis, yaxis };
    }

    const isInverted = (properties?.controls?.orientation?.position as ChartOrientationType) === "vertical";
    if (isInverted && yaxis) {
```

Reply:
I have updated this. Thanks!